core/conversion/converters/Weights.cpp

```diff
@@ -18,16 +17,33 @@ Weights::Weights() {
 Weights::Weights(ConversionCtx* ctx, float val) {
     this->num_input_maps = 1;
     this->num_output_maps = 1;
+
     this->data.type = nvinfer1::DataType::kFLOAT;
     float* buf = reinterpret_cast<float*>(malloc(1 * sizeof(float)));
     buf[0] = val;
     this->data.values = buf;
     this->data.count = 1;
     ctx->builder_resources.push_back(buf);
+    
     this->kernel_shape.nbDims = 1;
     this->kernel_shape.d[0] = 1;
```

Comment:
We need to add the shape initialization here too, also lets make this a scalar vs a 1d tensor